docker/vernemq/bin/vernemq.sh

```
@@ -0,0 +1,352 @@
+        echo "Permissions ok: Our pod $(hostname) belongs to StatefulSet ${statefulSetName} with ${numReplicas} replicas"
+    fi
+fi
+
+# Set up kubernetes node discovery
+start_join_cluster=0
+if env | grep "DOCKER_VERNEMQ_DISCOVERY_KUBERNETES" -q; then
+    # Let's set our nodename correctly
+    # https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#list-pod-v1-core
+    podList=$(k8sCurlGet "api/v1/namespaces/${DOCKER_VERNEMQ_KUBERNETES_NAMESPACE}/pods?labelSelector=${DOCKER_VERNEMQ_KUBERNETES_LABEL_SELECTOR}")
+    VERNEMQ_KUBERNETES_SUBDOMAIN=${DOCKER_VERNEMQ_KUBERNETES_SUBDOMAIN:-$(echo ${podList} | jq '.items[0].spec.subdomain' | tr '\n' '"' | sed 's/"//g')}
+    if [[ $VERNEMQ_KUBERNETES_SUBDOMAIN == "null" ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)